config/abilities/provider_any.rb

```diff
@@ -9,6 +9,10 @@
     end
 
     can :manage, user
+
+    # Overriding `can :manage, user` and `can :manage, User, :id => user.id`
+    cannot :update_permissions, User, &:admin?
```

Comment:
There is a bit of mess with abilities, because of the order in which they are executed. See https://issues.jboss.org/browse/THREESCALE-1326

As the member permissions setters belong to the `User` model, `can :manage, User` overrides the `cannot :update_permissions, User`